### ISSUE TITLE
torch.ifft(x, 1) 

### ISSUE THREAD

**Issue by WenBingo** [mingyuyng/Deep-JSCC-for-images-with-OFDM#7]
torch.ifft(input, signal_ndim, normalized=False) , 在pytorch手册中，signal_ndim表示信号的维度，但是作者您对一个4维度信号进ifft时候却写signal_ndim=1,  没搞懂

**Comment by mingyuyng** [comment]
Hi @WenBingo. Thanks for the question. This could be a potential error of the old code. Please check the updated code where torch.fft.ifft is used instead. The IFFT operation is always performed for OFDM symbols, which are 1-d complex signals.

**Comment by WenBingo** [comment]
thanks for your answear , i change code to "info_pilot = torch.fft.fft(info_pilot, 1)   info_pilot_hat = torch.cat(( info_pilot.real,  info_pilot.imag), -1)"  this way can run right
